framework/include/utils/RankFourTensor.h

```diff
@@ -300,19 +307,19 @@ class RankFourTensor
   void fillSymmetricIsotropicEandNuFromInputVector(const std::vector<Real> & input);
```

Comment:
Do we want to get rid of this function now (since its functionality is replaced by `fillSymmetricIsotropicEandNu()`? I imagine we have some code in the apps that use this function.